mise.toml

```diff
@@ -0,0 +1,2 @@
+[tools]
+elixir = "1.18.4"
```

Comment:
Please remove this, pretty sure it was added by a mistake.

Reply:
Yes, not sure how that happened, thanks.